scripts/e2e-test.sh

```diff
@@ -75,6 +82,16 @@ for dir in $TESTS; do
   fi
 done
 
+if [ -z "$test_failed" ] && [ -n "$RUN_LONG_TESTS" ]; then
+  for dir in $LONG_TESTS; do
+    cd "$SCRIPTDIR/../test/e2e/$dir"
+    if ! go test -v . -ginkgo.v -ginkgo.progress -timeout 0 ; then
```

Comment:
Does `ginko.progress` play well with redirected stdout & jenkins? Does it make sense here?

Reply:
Good point. I have looked into the jenkins log from previous runs and the output seems to look ok. So I think it can stay if it makes the output better for interactive use case.